### PR TITLE
Cross-plat cryptography article

### LINKED ISSUE #761: Document how to use cryptography in a cross-plat way
There seems to be some gotchas when working with crypto APIs on different platforms (can't think of anything specific, but seen stuff pop up on aliases and online). Some samples or examples could be really helpful for someone starting out with a crypto scenario on .NET Core.

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #761

### CODE REVIEW

File: docs/standard/security/cross-platform-cryptography.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+  AES-GCM works with 128, 192, and 256-bit keys.
+
+* Nonce Sizes
+
+  The <xref:System.Security.Cryptography.AesGcm> class supports only 96-bit (12-byte) nonces.
+
+* Tag Sizes
+
+  The <xref:System.Security.Cryptography.AesGcm> class supports creating or processing 96, 104, 112, 120, and 128-bit (12,, 13, 14, 15, and 16-byte) tags.
```
Comment (bartonjs) [inline docs/standard/security/cross-platform-cryptography.md:76]:
```suggestion
  The <xref:System.Security.Cryptography.AesGcm> class supports creating or processing 96, 104, 112, 120, and 128-bit (12, 13, 14, 15, and 16-byte) tags.
```

File: docs/standard/security/cross-platform-cryptography.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+
+| Scenario                                     | Windows | Linux | macOS |
+|----------------------------------------------|---------|-------|-------|
+| Empty                                        | ✔️     | ✔️    | ❌   |
+| One certificate, no private key              | ✔️     | ✔️    | ❌   |
+| One certificate, with private key            | ✔️     | ✔️    | ✔️   |
+| Multiple certificates, no private keys       | ✔️     | ✔️    | ❌   |
+| Multiple certificates, one private key       | ✔️     | ✔️    | ✔️   |
+| Multiple certificates, multiple private keys | ✔️     | ❌    | ✔️   |
+| Ephemeral loading                            | ✔️     | ✔️    | ❌   |
```
Comment (bartonjs) [inline docs/standard/security/cross-platform-cryptography.md:268]: Ooh, something to update: All of the Xs in the read PKCS12 and write PKCS12 tables are checkmarks in .NET 5.

Maybe they should get warning glyphs with a footnote that it's available in .NET 5 previews, then turn into checkmarks after net5 RTM?

File: docs/standard/security/cross-platform-cryptography.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+### X509Chain
+
+macOS doesn't support Offline CRL utilization, so `X509RevocationMode.Offline` is treated as `X509RevocationMode.Online`.
+
+macOS doesn't support a user-initiated timeout on CRL (Certificate Revocation List) / OCSP (Online Certificate Status Protocol) / AIA (Authority Information Access) downloading, so `X509ChainPolicy.UrlRetrievalTimeout` is ignored.
+
+## Additional resources
+
+[.NET Cryptography Model](cryptography-model.md)
+[.NET Cryptographic Services](cryptographic-services.md)
```
Comment (bartonjs) [inline docs/standard/security/cross-platform-cryptography.md:361]: Are these supposed to render side by side, or as a bulleted list?  (Currently they're SxS)

Comment (tdykstra) [reply docs/standard/security/cross-platform-cryptography.md:361]: Bulleted list, will update, thanks.

**Review by bartonjs — APPROVED** [review]

**Pull request merged**